app/assets/v2/js/pages/bounty_details2.js

```diff
@@ -370,13 +388,6 @@ Vue.mixin({
         activities.forEach(activity => {
           if (activity.metadata) {
             if (activity.metadata.new_bounty) {
-              // ETH
-              activity.metadata.new_bounty['token_value'] = activity.metadata.new_bounty.value_in_token / 10 ** decimals;
-              if (activity.metadata.old_bounty) {
-                activity.metadata.old_bounty['token_value'] = activity.metadata.old_bounty.value_in_token / 10 ** decimals;
-              }
-            } else {
-              // cross-chain
```

Comment:
are not going to jump and use this file in eth bounties?

Reply:
I'd want to do it via the backend where we streamline the response and then pass it to the frontend !
Activity will be in common format when it comes to frontend (irrespective of if it's ETH or cross-chain)